tests/unittest/llmapi/test_llm_args.py

```
@@ -3404,43 +3399,33 @@ def _pretrained_config(architectures, model_type):
+        assert DeepseekV3ForCausalLM.get_preferred_transceiver_runtime(
+        ) == "PYTHON"
 
-        End-to-end through _resolve_transceiver_runtime_auto with the real
-        model class and a GLM pretrained config.
-        """
+    def test_deepseek_resolves_auto_to_python_on_nixl(self):
+        """DeepseekV3ForCausalLM on NIXL adopts the Python transceiver from 'auto'."""
         from tensorrt_llm._torch.models.modeling_deepseekv3 import \
             DeepseekV3ForCausalLM
         args = TorchLlmArgs(
             model="/tmp/dummy_model",
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/dummy_model"

(S108)